mono/metadata/w32handle.h

```diff
@@ -10,20 +10,14 @@
 
 #ifdef HOST_WIN32
 #include <windows.h>
+#else
```

Comment:
All changes in this file are useless. Revert them.

Reply:
The point is to have one and only one definition of a constant. Not have ‘#include’ order change the meaning of things.

Reply:
It never have been a problem until now. If anything the 2 #include of mono/utils should be removed instead.